spec/controllers/comments_controller_spec.rb

```diff
@@ -0,0 +1,86 @@
+require 'rails_helper'
+require 'pry'
+
+
+describe CommentsController do
+
+  let(:user){create(:user)}
+
+  before :each do
```

Comment:
I see here you set the actual cookies values.

The way I handled needing to authenticate the user I just stubbed it out instead.